users/gurtatiLND/website/webpage.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="'en">
+    <head>
+      <meta charset="UTF-8">
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+      <meta name="author" content="Tatiana Gurova">
+    </head>
+    <body>
+        <header class="header">
+            <div class="header_logo">My Photo</div>
+            <div class="header_main">Links to the sections</div>
+        </header>
+        <main>
+            <section class="home">Home</section>
+            <section id="about" class="about sec-pad">About</section>
+            <section id="projects" class="projects sec-pad">Projects</section>
+            <section id="contact" class="contact sec-pad">Contact</section>
+        </main>
+        <footer class="main-footer">
+            <p>
+                @Copyright 2024. Made by <a href="https://github.com/gurtatiLND">Tatiana Gurova</a>.
```

Comment:
To replicate the copyright symbol, I'd recommend trying to look for HTML character entity reference. 

```html
<p> &copy; Copyright 2024.... </p>
```

Reply:
oh thank you! will do this